queue/index.js

```diff
@@ -0,0 +1,47 @@
+export class Queue {
+  constructor(app, actionTypes, key) {
+    this.app = app
+    this.actionTypes = actionTypes
+    this.key = key
+    this.data = []
+    this.processing = false
+  }
+
+  add(action, meta) {
+    this.data.push(action)
```

Comment:
Let’s add them together as `this.data.push([action, meta])`

Reply:
Yes this would be better!

Reply:
Fixed